CHANGELOG.md

```diff
@@ -65,6 +65,14 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+#### Style the `html` in your cookie banner
+
+We have removed the default font styles from the cookie banner Sass, to make it more obvious when classes/styles have been forgotten.
+
+If you are passing `html` into the cookie banner component, you must make sure you are applying the appropriate classes/styles to that HTML, such as adding the `govuk-body` class to any paragraph tags. You must do this to prevent your cookie banner displaying with unstyled text.
```

Comment:
```suggestion
If you're passing `html` into the cookie banner component, make sure you're applying the appropriate classes and styles to that HTML. For example, add the `govuk-body` class to any paragraph tags. You must do this to prevent your cookie banner displaying with unstyled text.
```
Wonder could we delete "to that HTML" from the second sentence? Or is there a risk some users might think it's the cookie banner component they have to apply classes and styles to?